### PR TITLE
Add UTs for hostfactory 'create token' command

### PR DESCRIPTION
**Pull request by sigalsax**
### What does this PR do?
Adds UTs for hostfactory 'create token' command

### What ticket does this PR close?
Resolves #JIRA

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

### CODE REVIEW

File: conjur/cli.py [hunk 1]
```diff
@@ -185,7 +185,7 @@ def handle_hostfactory_logic(cls, args: list = None, client=None):
             hostfactory_logic = HostFactoryLogic(client)
 
             days = args.duration_days if args.duration_days else 0
-            hours = args.duration_hours if args.duration_hours else 0
+            hours = args.duration_hours if args.duration_hours else 1
```
Comment (orenbm) [inline conjur/cli.py:188]: i'm guessing this is approved by Sapir?

**Review by orenbm — DISMISSED** [review]

File: conjur/cli.py (hunk 1 above)
Comment (sigalsax) [reply conjur/cli.py:188]: set this back to 0

Comment (orenbm) [reply conjur/cli.py:188]: what is the requirement from product?

**Review by eladkug — CHANGES_REQUESTED** [review]
left some comments

File: test/test_unit_hostfactory_controller.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            mock_hostfactory_controller.create_token(create_token_data=None)
+
+
+    def test_empty_host_factory_id_raises_correct_error(self):
+        mock_hostfactory_logic = HostFactoryLogic
+        mock_hostfactory_controller = HostFactoryController(mock_hostfactory_logic)
+        with self.assertRaises(MissingRequiredParameterException):
+            mock_hostfactory_controller.create_token(create_token_data=CreateTokenData(host_factory=None))
+
+    def test_empty_host_factory_duration_raises_correct_error(self):
```
[the remark this answers is not in the document]
Comment (sigalsax) [reply test/test_unit_hostfactory_controller.py:23]: specific flags. This is just the vanilla flow

**Review by eladkug — APPROVED** [review]
LGTM only for vanilla flow UT's , the rest of the test will be per flag

File: conjur/cli.py (hunk 1 above)
Comment (sigalsax) [reply conjur/cli.py]: we default to 1 (see create_token_data)
I moved this back to 0 because defaults shouldnt be set in the view

**Review by orenbm — APPROVED** [review]

**Pull request merged**